my-dsl.md

```diff
@@ -1,39 +1,54 @@
 # Language
 
-_What is the name of the language? Link the name to its webpage
-(if appropriate)._
+_What is the name of the language? Link the name to its webpage (if appropriate)._
+
+The DSL that I chose is Regualr Languages or Regex Expressions. The reason why I chose this DSL is because I have worked with them before in a prior internship. [java.util.regex regex link for java](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html)
 
 # Domain
 
 _Describe the language's domain in five words._
 
+External DSL applying pattern recognition 
+
 # Computational model
 
 _We don't yet have a great definition of the term "computational model".
 For now, try to come up with the clearest, most concise explanation of
 what happens when a program in your DSL runs._
 
+Regular Langages analyze and find patterns in strings of code through recursive pattern analysis where the pattern is defined by the user. For instance lets say the user wanted to find a phone number from a list of peoples personal information that may include a bunch of other information that is not impornta to the user. The user could define a regular expression to find phone numbers. This DSL would start by recusivly running through the list and it would itentify anything in the list that matches the pattern of a phone number (+1(123) 456-6789).
```

Comment:
I like this explanation - everything I know about regex is how powerful being able to find patterns is. It's a really helpful ability! And the example was very clear and concise.